jupyterhub/templates/hub/configmap.yaml

```diff
@@ -213,6 +210,9 @@ data:
     {{- end }}
   {{- end }}
 
+  {{- if .Values.scheduling.userScheduler.enabled }}
+  singleuser.scheduler-name: "user-scheduler"
+  {{- end }}
```

Comment:
This is the way we pass through userScheduler enabled to be true or false pretty much. One could argue that it should be declared as true/false instead and let the jupyterhub_config.py figure out what scheduler-name to set based on that.